.storybook/stories/SelectableTableBatchActions/ModalAddTrackingNumber.tsx

```diff
@@ -9,13 +9,16 @@ import { DatePicker } from '@ag.ds-next/react/date-picker';
 export type ModalAddTrackingNumberProps = {
 	isOpen: boolean;
 	onConfirm: () => void;
-	onDismiss: () => void;
+	/** @deprecated use `onClose` instead */
```

Comment:
This is just a story, so we can remove this

Reply:
removed